hibernate-core/src/main/java/org/hibernate/tool/schema/internal/ColumnDefinitions.java

```diff
@@ -249,13 +258,49 @@ private static String normalize(String typeName) {
 		}
 	}
 
+	private static boolean isExpected(String columnDefinition, ColumnInformation columnInformation, Dialect dialect) {
+		String expectingTypeName = extractTypeName( columnDefinition );
+		if ( dialect instanceof OracleDialect && expectingTypeName.equals( "timestamp with time zone" ) ) {
+			// Fix: found [timestamp (Types#UNKNOWN(-101))], but expecting [timestamp(6) with time zone (Types#TIMESTAMP_UTC)]
```

Comment:
It works fine previously because `stripArgs` just keep string before `(`, now it drop `(*)` for keep `unsigned` from `int(6) unsigned`.